src/mame/video/powervr2.cpp

```diff
@@ -2073,6 +2080,41 @@ void powervr2_device::process_ta_fifo()
 						ts->svert = rd->verts_size;
 						ts->evert = rd->verts_size + 3;
 
+						rd->verts_size += 4;
+					}
+				}
+				else
+				{
+					if (rd->verts_size <= (MAX_VERTS - 6) && grp->strips_size < MAX_STRIPS)
+					{
+						strip *ts;
+						vert *tv = &rd->verts[rd->verts_size];
```

Comment:
It’s better to declare variables at the point where you initialise them.  Also, you can make `tv` a constant pointer since its value shouldn't change (`vert *const tv = ...`) and you can make `ts` a reference since you don’t try to index it like an array and it should never be null.

Reply:
This section was copied verbatum from the above texture section, but the point still stands. I'll clean this code up a bit.

Reply:
Cleaned up.